clients/rest-client-generator/src/test/java/uk/gov/justice/services/clients/rest/generator/strategy/ClientMediaTypeGenerationTest.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.justice.services.clients.rest.generator.strategy;
+
+import static java.util.Collections.emptyList;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static uk.gov.justice.services.adapters.test.utils.builder.MappingBuilder.defaultMapping;
+import static uk.gov.justice.services.adapters.test.utils.builder.MappingDescriptionBuilder.mappingDescriptionWith;
+
+import uk.gov.justice.raml.common.mapper.ActionMapping;
+
+import java.util.List;
+
+import org.junit.Test;
+import org.raml.model.ActionType;
+import org.raml.model.MimeType;
+
+public class ClientMediaTypeGenerationTest {
+
+    private static final String DESCRIPTION = mappingDescriptionWith(defaultMapping()).build();
+
+    @Test
+    public void shouldReturnListOfActionMappings() throws Exception {
```

Comment:
can we update the test names so it is clear that we are expecting empty/null values.